plutus-wallet-api/ledger/Ledger/Typed/Scripts.hs

```diff
@@ -42,35 +46,32 @@ type ValidatorType (a :: Type) = DataType a -> RedeemerType a -> Validation.Pend
 
 type WrappedValidatorType = Data -> Data -> Data -> ()
 
--- | A typed validator script with its 'Validator' and 'Address'.
-data ScriptInstance (a :: Type) where
-    Validator ::
-        ScriptType a
-        => CompiledCode (ValidatorType a)
-        -> CompiledCode (ValidatorType a -> WrappedValidatorType)
-        -> Validator
-        -> Addr.Address
-        -> ScriptInstance a
+-- | A typed validator script with its 'ValidatorScript' and 'Address'.
+data ScriptInstance (a :: Type) =
```

Comment:
Why the change here?

Reply:
We don't use the `CompiledCode` values anywhere (also I wanted to get the JSON instances so that I can save the instance with a `checkpoint` in the future contract)

Reply:
Hmm okay. I guess it's just a phantom type now, which seems fine.